lib/djvViewApp/PlaybackSpeedWidget.cpp

```diff
@@ -148,6 +166,49 @@ namespace djv
                         }
                     }
                 });
+
+            p.useCustomSpeedButton->setClickedCallback(
+                [weak]
+                {
+                    if (auto widget = weak.lock())
+                    {
+                        widget->_doSpeedCallback(widget->_p->customSpeed);
+                    }
+                });
+
+            p.customSpeedLineEdit->setTextEditCallback(
+                [weak](const std::string& value, UI::TextEditReason)
+                {
+                    if (auto widget = weak.lock())
+                    {
+                        if (std::stof(value) >= 1)
+                        {
+                            try
+                            {
+                                widget->_p->customSpeed = Time::Speed(std::stoi(value));
+                            }
+                            catch (const std::exception&)
+                            {
+                                std::stringstream ss;
+                                ss << "Cannot parse the value.";
+                                widget->_log(ss.str(), LogLevel::Error);
+                            }
+                        }
+                        else if (1 > std::stof(value) > 0)
```

Comment:
I'm getting a compiler warning about this line (warning C4804:  '>': unsafe use of type 'bool' in operation), I think it needs an "&&" operator:

    std::stof(value) > 0.F && std::stof(value) < 1.F